package.json

```diff
@@ -145,12 +145,16 @@
       "<rootDir>/node_modules/react-native/",
       "<rootDir>/lib/",
       "<rootDir>/test/react/",
-      "<rootDir>/fb-www/"
+      "<rootDir>/fb-www/",
+      "<rootDir>/test/debugger",
+      "<rootDir>/scripts"
     ],
     "testEnvironment": "jsdom",
     "testMatch": [
       "**/test/react/*-test.js",
-      "**/fb-www/**/*-test.js"
+      "**/fb-www/**/*-test.js",
+      "**/test/debugger/*-test.js",
+      "**/*debugger*.js"
```

Comment:
I think this might be too general for a testMatch, maybe `**/scripts/*.test.js`, or move that script to somewhere under the `test/` directory.